api/cueSearch/services/cardTemplate.py

```diff
@@ -123,3 +125,24 @@ def getCardTemplateById(templateId: int):
             logging.error("Error while get card template by Id %s", str(ex))
             res.update(False, "Error occured while getting template by id")
         return res
+
+    @staticmethod
+    def verifyCardTemplate(param: dict):
+        res = ApiResponse()
+        response = []
+        delimiter = "+-;"
+        try:
+            sqls = (
+                Template(param["templateSql"]).render(Context(param)).split(delimiter)
```

Comment:
`Context(param)` - don't pass param here, instead create a sample values against which user template will be tested.
Like a test case kindof